static/app/views/replays/replaySearchBar.tsx

```diff
@@ -121,6 +123,20 @@ function ReplaySearchBar(props: Props) {
       maxMenuHeight={500}
       hasRecentSearches
       fieldDefinitionGetter={getReplayFieldDefinition}
+      onSearch={(query: string) => {
+        props?.onSearch?.(query);
+        const conditions = new MutableSearch(query);
+        const searchKeys = conditions.tokens
+          .map(({key}) => key)
+          .filter(v => typeof v === 'string') as string[];
+
+        if (searchKeys.length > 0) {
+          trackAdvancedAnalyticsEvent('replay.search', {
```

Comment:
This tracks to amplitude?

Reply:
@eliashussary @billyvg No it won't, use a comma delineated string instead here